### PR TITLE
Add Workflows Working Group list of WGs

### PR DESCRIPTION
**Pull request by dibyom**
/kind misc

### CODE REVIEW

**Comment by sbwsg** [comment]
I'm happy to help facilitate as well.

/approve

**Comment by vdemeester** [comment]
@dibyom is the worflows WG the same as pipeline-as-code WG that was discussed at some point ?

**Comment by chmouel** [comment]
@vdemeester yep!

/lgtm

would be happy to facilitate this as well,

File: working-groups.md [hunk 1]
```diff
@@ -313,3 +314,20 @@ This is the working group for [`tektoncd/chains`](https://github.com/tektoncd/ch
...
+## Workflows
+
+Topics for this WG include the experimental [Tekton Workflows](https://github.com/tektoncd/experimental/tree/main/workflows), [Pipelines as Code](https://github.com/openshift-pipelines/pipelines-as-code), [Remote Resolution](https://github.com/tektoncd/community/blob/main/teps/0060-remote-resource-resolution.md) etc.
+
+| Artifact                   | Link                       |
+| -------------------------- | -------------------------- |
+| Forum                      | [tekton-dev@](https://groups.google.com/forum/#!forum/tekton-dev) |
+| Community Meeting VC       | [meet.google.com/jwa-pxmv-vpi](https://meet.google.com/jwa-pxmv-vpi) |
+| Community Meeting Calendar | Tuesdays every week, 10am -10:30am ET <br>[Calendar](https://calendar.google.com/event?action=TEMPLATE&tmeid=NDl0bzN1OWl2dWxlMXBpNGFnaDhjM2N2NjRfMjAyMTA5MDdUMTQwMDAwWiBnb29nbGUuY29tX2Qzb3Zjdm8xcDMyMTloOTg5NTczdjk4Zm5zQGc&tmsrc=google.com_d3ovcvo1p3219h989573v98fns%40group.calendar.google.com&scp=ALL) |
+| Meeting Notes              | [Notes](https://docs.google.com/document/d/1UVPSCDyNO-TzEFSv8jrqrEOF_FmV8NFuXncFm1gwmeY/edit) |
```
Comment (chmouel) [inline working-groups.md:327]: I think this is the wrong link going to chains

File: working-groups.md [hunk 2]
```diff
@@ -313,3 +314,20 @@ This is the working group for [`tektoncd/chains`](https://github.com/tektoncd/ch
 | -----------------------------------------------------------    | ----------     | ------- | ------------------------------------------------  |
 | <img width="30px" src="https://github.com/dlorenc.png">        | Dan Lorenc     | Google  | [dlorenc](https://github.com/dlorenc) |
 | <img width="30px" src="https://github.com/priyawadhwa.png">    | Priya Wadhwa   | Google  | [priyawadhwa](https://github.com/priyawadhwa)     |
+
+## Workflows
```
Comment (chmouel) [inline working-groups.md:318]: I don't mind calling it workflow but we may want to call it like this then for the slack channel and meeting invite to not confuse users,

**Comment by khrm** [comment]
I am also happy to facilitate this.

File: working-groups.md (hunk 1 above)
Comment (dibyom) [reply working-groups.md:327]: 🤦‍♂️

**Comment by dibyom** [comment]
> @dibyom is the worflows WG the same as pipeline-as-code WG that was discussed at some point ?

Yes.

File: working-groups.md (hunk 2 above)
Comment (chmouel) [reply working-groups.md:318]: Yes i have just done it!

**Review by vdemeester — APPROVED** [review]

**Comment by chmouel** [comment]
@vdemeester pulled the /approve triggers too quickly :-p the link is still going to the wrong document

**Comment by dibyom** [comment]
> @vdemeester pulled the /approve triggers too quickly :-p the link is still going to the wrong document

Let me fix that!

**Pull request merged**